content/blog/simplify-embedded-projects-with-esp-bsp/index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Simplify Your Embedded Projects with ESP-BSP"
+date: 2024-06-18
+showAuthor: false
+authors:
+    - "juraj-michalek"
+tags: ["Embedded Systems", "ESP32", "ESP32-S3", "Espressif", "BSP"]
+---
+
+## Introduction
+
+Are you a maker or an embedded systems enthusiast looking to create applications that work across different development boards with Espressif SoCs like ESP32 or ESP32-S3? Whether you’re using the [ESP-WROVER-KIT](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/hw-reference/esp32/get-started-wrover-kit.html), [M5Stack-CoreS3](https://docs.m5stack.com/en/core/CoreS3), [ESP32-S3-BOX-3](https://github.com/espressif/esp-box/blob/master/docs/hardware_overview/esp32_s3_box_3/hardware_overview_for_box_3.md), or other compatible boards, the ESP Board Support Package (ESP-BSP) makes your life easier. In this article, we’ll walk you through how to get started with ESP-BSP, enabling you to focus on your project’s functionality without worrying about hardware differences.
+
+## What is ESP-BSP?
+
+[ESP-BSP](https://github.com/espressif/esp-bsp) is a collection of Board Support Packages featuring Espressif’s or M5Stack development boards. The set can be extended to any board with Espressif’s chip. It provides a convenient way to start a project for a specific development board without the need to manually look for drivers and other details. By using ESP-BSP, you can:
```

Comment:
Doesn't have to be changed:

Should we also mention here that esp-bsp works with esp-idf only? that is, it does not work with arduino?

Reply:
Good point. Reformulated. I made it explicit, that it's for ESP-IDF based projects.